superdesk/auth_server/models.py

```diff
@@ -52,7 +53,11 @@ def get_allowed_scope(self, scope):
 
 def query_client(client_id):
     clients_service = superdesk.get_resource_service('auth_server_clients')
-    client_data = clients_service.find_one(req=None, _id=ObjectId(client_id))
+    try:
+        client_data = clients_service.find_one(req=None, _id=ObjectId(client_id))
+    except InvalidId:
```

Comment:
Why is the error silently ignored? IMHO, at least a `logger.error` should be done here.

Reply:
Log message was added.